test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs

```diff
@@ -137,5 +137,37 @@ public static void RemoveSetting(XDocument doc, string key)
                 item.Remove();
             }
         }
+
+        // Add NetStandard.Library and NetCorePlatforms to the feed and save the file.
+        public void AddNetStandardFeeds()
+        {
+            string[] feedlist = GetNetstandrdAndNetcorePackages();
+            var section = GetOrAddSection(XML, "packageSources");
+            AddEntry(section, "NetStandardLibrary", feedlist[0]);
+            AddEntry(section, "NetCorePlatforms", feedlist[1]);
+
+            Save();
+        }
+
+        private static string[] GetNetstandrdAndNetcorePackages()
+        {
+            string[] list = new string[2];
+            DirectoryInfo currentDir = new DirectoryInfo(Directory.GetCurrentDirectory());
+            while (currentDir != null)
+            {
+                if (currentDir.GetFiles().Any(e => e.Name.Equals("NuGet.sln", StringComparison.OrdinalIgnoreCase)))
```

Comment:
This gets all the files and then looks in that set for one specific file.

Why not simply look for one specific file?
```C#
if (File.Exists(Path.Combine(currentDir.FullName, "NuGet.sln")))
```

Reply:
Thanks. Changed.